lib/core/utils/provider_initializer.dart

```diff
@@ -90,7 +90,7 @@ class ProviderInitializer {
                 FetchBlogsUseCase(ArticleRepositoryImpl(newsApiClient))),
       ),
       BlocProvider(
-          create: (context) => EventsPageCubit(
+          create: (context) => EventsCubit(
```

Comment:
The Cubit name changed from `EventsPageCubit` to `EventsCubit` might indicate a change in the responsibility or behavior of the Cubit that should be reviewed thoroughly if it was intended.